### PR TITLE
Fixes absolute path issue for increment_build_number and increment_ve…

### PR DESCRIPTION
**Pull request by xfreebird**
Fixes absolute path issue for increment_build_number and increment_version_number #747

### CODE REVIEW

File: lib/fastlane/actions/get_build_number.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ def self.run(params)
         # More information about how to set up your project and how it works:
         # https://developer.apple.com/library/ios/qa/qa1827/_index.html
 
-        folder = params[:xcodeproj] ? File.join('.', params[:xcodeproj], '..') : '.'
+        folder = params[:xcodeproj] ? File.join(params[:xcodeproj], '..') : '.'
```
Comment (kaoskorobase) [inline lib/fastlane/actions/get_build_number.rb]: I think using `File.dirname(params[:xcodeproj])` would be clearer here.

Comment (xfreebird) [inline lib/fastlane/actions/get_build_number.rb]: you give it let say ```/path/to/mysuper.xcodeproj``` in the end it needs ```/path/to```.
That is why there is added '..'

**Comment by KrauseFx** [comment]
Perfect, thanks @xfreebird :+1:

File: lib/fastlane/actions/get_build_number.rb (hunk 1 above)
Comment (kaoskorobase) [inline lib/fastlane/actions/get_build_number.rb]: Yes, that's essentially what `File.dirname` returns, the directory part of a path.

Comment (xfreebird) [inline lib/fastlane/actions/get_build_number.rb]: Haven't thought about using that, you could take over and make a new PR, this one is already merged.

**Pull request merged**